di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json

```diff
@@ -2395,6 +2395,182 @@
         ]
       }
     },
+    {
+      "criType": "DWP KBV (Stub)",
+      "label": "Alice Parker (Valid) DWP KBV",
+      "payload": {
+        "name": [
+          {
+            "nameParts": [
+              {
+                "value": "Aliçe",
```

Comment:
Are the "c" here and the "e" in parker intentionally special? I couldn't see it other examples

Reply:
This data was duplicated from the HMRC KBV data above it. Looks like the Experian KBV / NINO / BAV use it with this "ç" as well

Reply:
I think this is deliberate to test name correlation 😄